.github/workflows/jetbrains-update-plugin-platform-template.yml

```diff
@@ -0,0 +1,76 @@
+on:
+    workflow_call:
+        inputs:
+            pluginName:
+                description: Name of the plugin.
+                type: string
+                required: true
+            pluginId:
+                description: ID of the plugin in lowercase and without spaces.
+                type: string
+                required: true
+            xpath:
+                description: Xpath for the latest platform version in https://www.jetbrains.com/intellij-repository/snapshots
+                type: string
+                required: true
+            gradlePropertiesPath:
+                description: Path for the gradle.properties file of the plugin.
+                type: string
+                required: true
+jobs:
+    update-plugin-platform:
+        name: Update Platform Version from ${{ inputs.pluginName }}
+        runs-on: ubuntu-latest
+        env:
+            SNAPSHOTS_HTML_FILENAME: snapshots.html
+        steps:
+            - name: Checkout Repository
+              uses: actions/checkout@v3
+            - name: Save the snapshots page to an HTML file
+              run: curl -sL https://www.jetbrains.com/intellij-repository/snapshots > ${{ env.SNAPSHOTS_HTML_FILENAME }}
+            - name: Get Latest Platform Version
+              uses: QwerMike/xpath-action@v1
+              id: latest-version
+              with:
+                  filename: ${{ env.SNAPSHOTS_HTML_FILENAME }}
+                  expression: ${{ inputs.xpath }}
+            - name: Get Current Platform Version
+              id: current-version
+              run: |
+                  rm ${{ env.SNAPSHOTS_HTML_FILENAME }}
+                  CURRENT_PLATFORM_VERSION=$(cat ${{ inputs.gradlePropertiesPath }} | grep platformVersion= | sed 's/platformVersion=//')
+                  echo "Current platform version: $CURRENT_PLATFORM_VERSION"
+                  echo "Latest platform version:  ${{ steps.latest-version.outputs.result }}"
+                  echo "::set-output name=result::$CURRENT_PLATFORM_VERSION"
+            - name: Print Result
+              run: |
+                  echo "Current platform version: ${{ steps.current-version.outputs.result }}"
+                  echo "Latest platform version:  ${{ steps.latest-version.outputs.result }}"
+            - name: Create Pull Request
+              if: ${{ steps.latest-version.outputs.result != steps.current-version.outputs.result }}
+              uses: peter-evans/create-pull-request@v4
+              with:
+                  title: "Update Platform Version from ${{ inputs.pluginName }}"
+                  body: |
+                      ## Description
+                      This PR updates the Platform Version from ${{ inputs.pluginName }} to the latest version.
+
+                      ## How to test
+                      1. Ensure you have the latest JetBrains Gateway installed: https://www.jetbrains.com/remote-development/gateway/
```

Comment:
Is it all relevant for backend plugin update? Do we really need to deal with backend plugin in this PR?

Reply:
I thought about making different "How to test" messages for each component. But then it would remove 50% of the usefulness of the Reusable Workflow. I don't think it would take much time if the reviewer tested both components when updating some of them.

But let me update it to have individual "How to test" per component.
(As I also have to fix the [hardcoded major version issue](https://github.com/gitpod-io/gitpod/pull/11657#discussion_r930321611), I'll review the need for the template.yml.)

Reply:
Yeah, it was too much for testing the backend plugin.
I've updated it to have individual "How to test" per component now.
What do you think?